openshift-storage-libs/openshiftstoragelibs/openshift_ops.py

```diff
@@ -464,14 +464,15 @@ def oc_create_tiny_pod_with_volume(hostname, pvc_name, pod_name_prefix='',
 
 
 def oc_delete(
-        ocp_node, rtype, name, raise_on_absence=True, collect_logs=False,
-        skip_res_validation=True, is_force=False):
+        ocp_node, rtype, name=None, label=None, raise_on_absence=True,
+        collect_logs=False, skip_res_validation=True, is_force=False):
     """Delete an OCP resource by name
 
     Args:
         ocp_node (str): Node on which the ocp command will run.
         rtype (str): Name of the resource type (pod, storageClass, etc).
```

Comment:
`rtype` should be optional now. Function should work if any of the parameter `rtype` or `label` is provided and raise exception in case `None` is provided.

Reply:
`rtype` need to there, for `rtype=pvc` we will have a parameter `label` to delete